src/hal/i_components/biquad.icomp

```diff
@@ -122,8 +122,8 @@ Biquad_CalcCoeffs(void *arg, unsigned long period)
 {
 struct inst_data *ip = arg;
 
-    double                      sampleRate, w0, alpha;
-    double                      b0, b1, b2, a0, a1, a2;
+    hal_float_t sampleRate, w0, alpha;
```

Comment:
in line 132 "period" is implicitly converted to hal_float_t. A typecast should be added there.